src/controllers/user.controller.js

```diff
@@ -1,10 +1,10 @@
-import BaseRepository from '../repository/base.repository';
-import responseGenerator from '../helpers/responseGenerator';
-import utility from '../helpers/utils';
+import BaseRepository from "../repository/base.repository";
+import responseGenerator from "../helpers/responseGenerator";
+import utility from "../helpers/utils";
 
-import db from '../database/models';
```

Comment:
Double import of db file, kindly fix it.

Reply:
Thank you